mne/viz/eyetracking/heatmap.py

```diff
@@ -0,0 +1,157 @@
+# Authors: Scott Huberty <seh33@uw.edu>
+#
+# License: BSD-3-Clause
+
+import numpy as np
+from scipy.ndimage import gaussian_filter
+
+
+from ..utils import plt_show
+from ...utils import _ensure_int, _validate_type, logger, fill_doc
+
+
+@fill_doc
+def plot_gaze(
+    epochs,
+    width,
+    height,
+    *,
+    sigma=25,
+    cmap=None,
+    alpha=1.0,
+    vlim=(None, None),
+    axes=None,
+    show=True,
+):
+    """Plot a heatmap of eyetracking gaze data.
+
+    Parameters
+    ----------
+    epochs : instance of Epochs
+        The :class:`~mne.Epochs` object containing eyegaze channels.
+    width : int
+        The width dimension of the plot canvas. For example, if the eyegaze data units
+        are pixels, and the participant screen resolution was 1920x1080, then the width
+        should be 1920.
+    height : int
+        The height dimension of the plot canvas. For example, if the eyegaze data units
+        are pixels, and the participant screen resolution was 1920x1080, then the height
+        should be 1080.
+    sigma : float | None
+        The amount of Gaussian smoothing applied to the heatmap data (standard
+        deviation in pixels). If ``None``, no smoothing is applied. Default is 25.
+    %(cmap)s
+    alpha : float
+        The opacity of the heatmap (default is 1).
+    %(vlim_plot_topomap)s
+    %(axes_plot_topomap)s
+    %(show)s
+
+    Returns
+    -------
+    fig : instance of matplotlib.figure.Figure
+        The resulting figure object for the heatmap plot.
+
+    Notes
+    -----
+    .. versionadded:: 1.6
+    """
+    from mne import BaseEpochs
+    from mne._fiff.pick import _picks_to_idx
+
+    _validate_type(epochs, BaseEpochs, "epochs")
+    _validate_type(alpha, "numeric", "alpha")
+    _validate_type(sigma, ("numeric", None), "sigma")
+    _ensure_int(width, "width")
+    _ensure_int(width, "height")
```

Comment:
```suggestion
    width = _ensure_int(width, "width")
    height = _ensure_int(width, "height")
```

Contrary to `_validate_type` which returns None, `_ensure_int` will also cast to native python integer type.

Reply:
Thank you!